app/routes/product-details.$productSlug/route.tsx

```diff
@@ -144,6 +144,7 @@ export default function ProductDetailsPage() {
                                     title: section.title!,
                                     content: section.description ? (
                                         <div
+                                            className={styles.additionalInfoDescription}
```

Comment:
Is this an additional design tweak, or is it somehow related to `overflow: visible`?

Reply:
Previously, content and padding were animated separately. It looked smooth but was still noticable because the timing-funciton is not linear. Padding expands from 0 to 25px, and content expands from 0 to auto.
I made that padding part of the item content, so that everything is animated together.

Reply:
This makes default styling of accordion a bit sad:
<img width="401" alt="image" src="https://github.com/user-attachments/assets/2e8ba96b-3c8d-4a2f-acf9-7f2956826f00">

Now padding needs to be added externally every time the component is used.
How about adding another wrapper or a spacer div into the component itself?